### PR TITLE
Set the auth private key size to be 2048 bits

### PR DESCRIPTION
**Pull request by mprahl**
Clients such as Fedora 33 have a default OpenSSL configuration
that doesn't allow the usage of 1024 bit keys.

See more information at:
https://fedoraproject.org/wiki/Changes/StrongCryptoSettings2

closes #8317

### CODE REVIEW

**Comment by mprahl** [comment]
FYI, I did not run the tests locally since it wasn't immediately obvious how to run them. If a maintainer could verify them locally, that'd be appreciated.

**Comment by ipanova** [comment]
@mprahl thank you for the PR and sorry for the delayed answer. Please open an upstream issue and reference it in the commit message in the format of ` closes #1234 ` where 1234 is the issue number. https://pulp.plan.io/issues/new

**Comment by ipanova** [comment]
you need to add this new config option to the Security stanza https://github.com/pulp/pulp/blob/2-master/server/etc/pulp/server.conf#L110

**Comment by mprahl** [comment]
Thanks for the review @ipanova. Could you please take another look?

**Comment by mprahl** [comment]
@ipanova I've change it to hard code 2048 bit as previously discussed.

**Comment by bmbouter** [comment]
I can't speak to its backwards compatibility so I won't LGTM myself, but if it is fully backwards compatible I'm +1 on merging and releasing in a 2.21.z.

**Review by ipanova — APPROVED** [review]

**Pull request merged**
